packages/lib/models/Setting.ts

```diff
@@ -1121,7 +1121,24 @@ class Setting extends BaseModel {
 				storage: SettingStorage.File,
 				isGlobal: true,
 			},
-
+			imageResizing: {
+				value: 'alwaysAsk',
+				type: SettingItemType.String,
+				section: 'note',
+				isEnum: true,
+				public: true,
+				appTypes: [AppType.Mobile, AppType.Desktop],
+				label: () => _('Image resizing:'),
```

Comment:
[**Very minor/Feel free to ignore**]

Consider adding more context here — users that haven't encountered the "Do you want to resize this image" dialog might not know what this setting does. Other options for this label might be `Downsize large images`, or `Automatically shrink large images`.

Alternatively a `description` option could be added,
```suggestion
				label: () => _('Image resizing:'),
				description: () => _('Shrink large images before adding them to notes to save storage space.'),
```